kythe/release/README.md

```diff
@@ -146,13 +150,19 @@ Read `extractors/javac-wrapper.sh` for more details on its usage.
 
     extractors/cxx_extractor -Ithird_party/include some.cc
 
+    cd $GOPATH/src
+    # Build the tests along with the package.
+    go test -i github.com/MyOrg/my-package
+    $KYTHE_ROOT_DIRECTORY/extractors/go_extractor github.com/MyOrg/my-package \
+      --output $KYTHE_OUTPUT_DIRECTORY/extracted.kzip
+
 ## Indexers
 
 `indexers/cxx_indexer`, `indexers/go_indexer`, and `indexers/java_indexer.jar`
-analyze the .kindex files produced by the extractors and emit a stream of
-protobuf wire-encoded facts (entries) that conform to https://kythe.io/schema.
-The output stream can be processed by many of the accompanying binaries in the
-`tools/` directory.
+analyze the .kindex  or .kzip files produced by the extractors and emit a stream
```

Comment:
We're in the (slow) process of removing .kindex files, it should be fine to just remove their mention here.

Reply:
I changed this one sentence. 

Do the java/c++ extractors output kzip by default or with some flag? It could be a little confusing to have .kindex files mentioned everywhere else but here. If you could give me the commands for outputting .kindex files with the other extractors, I could move all of the examples to .kzip.